explorer/ui/src/containers/DagContainer.ts

```diff
@@ -51,40 +55,94 @@ export class DagStep {
   last = this.step(() => 0);
 }
 
+enum SubscribeState {
+  ON,
+  CLOSE
+}
+
 export class DagContainer {
   @observable blocks: BlockInfo[] | null = null;
   @observable selectedBlock: BlockInfo | undefined = undefined;
   @observable depth = 10;
   @observable maxRank = 0;
-  @observable toggleValidatorsList: boolean = false;
+  @observable validatorsListToggleStore: ToggleStore = new ToggleStore(false);
   @observable lastFinalizedBlock: BlockInfo | undefined = undefined;
   @observable eventsSubscriber: Subscription | null = null;
+  @observable subscribeToggleStore: ToggleStore = new ToggleStore(true);
 
   constructor(
     private errors: ErrorContainer,
     private casperService: CasperService
-  ) {}
+  ) {
+    // so that change of subscribeToggleStore can trigger `setUpSubscriber`
+    autorun(() => this.setUpSubscriber(this.subscribeToggleStore.isPressed), {
+      delay: 100
+    });
+  }
 
   get minRank() {
     return Math.max(0, this.maxRank - this.depth + 1);
   }
 
-  toggleShowValidators() {
-    this.toggleValidatorsList = !this.toggleValidatorsList;
-  }
-
   get hasBlocks() {
     return this.blocks ? this.blocks.length > 0 : false;
   }
 
+  get isLatestDag() {
+    return this.maxRank === 0;
+  }
+
+  private get _subscriberState(): SubscribeState {
+    if (this.eventsSubscriber && !this.eventsSubscriber.closed) {
+      return SubscribeState.ON;
+    } else {
+      return SubscribeState.CLOSE;
```

Comment:
```suggestion
      return SubscribeState.OFF;
```
`CLOSE` is a verb, it sounds more like an action than a state. Or maybe `CLOSED`?

Reply:
Done. Thank you.